src/app/app.js

```diff
@@ -0,0 +1,36 @@
+angular.module('graffio', [
+  'graffio.signupController',
+  'graffio.loginController',
+  'graffio.clickController',
+  'ui.router'
+])
+.config(function($stateProvider, $urlRouterProvider) {
+  var rootRef = new Firebase('https://dazzling-heat-2465.firebaseio.com/web/uauth');
+  var user = rootRef.getAuth();
+  if (!user) {
+    $urlRouterProvider.otherwise("/login");
```

Comment:
Just routing particular path, '.otherwise()' may not be the optimal solution there, but wasn't sure what the url would be for someone clicking on the icon in the browser, so wanted to make sure we caught all possibilities until we have a better solution.  

Logic was simply: on first click, we should check auth status, if they're not auth'd send them to login.